docs/tutorials/reuse-ssh-connections.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Reuse SSH Connections
```

Comment:
Currently, this tutorial doesn't call out the supported platforms at all; it needs to be clarified somehow. An alert doesn't make much sense, and including it in a single subheading is also illogical. With that, it seems to me like including the OS in the title is the best option. The OS shouldn't be included in the file name though.